quinn-proto/Cargo.toml

```diff
@@ -30,6 +30,8 @@ ring = ["dep:ring"]
 platform-verifier = ["dep:rustls-platform-verifier"]
 # Configure `tracing` to log events via `log` if no `tracing` subscriber exists.
 log = ["tracing/log"]
+# Enable rustls logging
+rustls-log = ["rustls/logging"]
```

Comment:
Should this be `rustls?/logging`?

Reply:
Good catch! Thanks, I'm still pretty new to Rust. Updated.

Reply:
Seems like you only updated one of them?

Reply:
So I did. Apologies. This time, for sure?